### PR TITLE
Search results page

### PR DESCRIPTION
**Pull request by anubra266**
Resolves EveripediaNetwork/issues#215

 
CleanShot 2022-04-14 at 18 19 47@2x

CleanShot 2022-04-14 at 18 20 38@2x

### CODE REVIEW

File: src/pages/search/[query].tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        }
+      },
+    )
+  }, [query])
+  const { articles, categories } = results
+  const totalResults = articles.length + categories.length
+
+  const articleList = articles.map(article => {
+    return (
+      <Flex key={article.id} bg="white" rounded="lg" shadow="lg" p="4">
```
Comment (kesar) [inline src/pages/search/[query].tsx:49]: for this cards you should use something similar to https://github.com/EveripediaNetwork/ep-ui/pull/165

its missing avatar, address, category.. for content we have a wrapper util that gets summary or content, check how others are implemented

**Comment by kesar** [comment]
also link in search suggestion doesnt seem to work on click

File: src/components/Activity/ActivityCard.tsx [hunk 2]
```diff
@@ -22,8 +22,8 @@ interface ActivityCardProps {
   title: string
   brief: string
   editor: string
-  lastModTimeStamp: string
-  wiki: Wiki
+  lastModTimeStamp?: string
+  wiki: Omit<Wiki, 'metadata' | 'version' | 'language'>
```
Comment (kesar) [inline src/components/Activity/ActivityCard.tsx:26]: typescript master 👍🏻

**Review by kesar — APPROVED** [review]

File: src/components/Activity/ActivityCard.tsx (hunk 2 above)
Comment (anubra266) [reply src/components/Activity/ActivityCard.tsx:26]: Thanks 😆

**Pull request merged**